src/app/pages/rpc-stx-transfer-sip9-nft/rpc-stx-transfer-sip9-nft-container.tsx

```diff
@@ -0,0 +1,75 @@
+import { Outlet, useNavigate } from 'react-router-dom';
+
+import { isDefined } from '@leather.io/utils';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { StacksNonceLoader } from '@app/components/loaders/stacks-nonce-loader';
+import { StxBalanceLoader } from '@app/components/loaders/stx-balance-loader';
+import { StacksFeeEditorProvider } from '@app/features/fee-editor/stacks/stacks-fee-editor.provider';
+import { NonceEditorProvider } from '@app/features/nonce-editor/nonce-editor.provider';
+import { useRpcTransactionRequest } from '@app/features/rpc-transaction-request/use-rpc-transaction-request';
+import { useBreakOnNonCompliantEntity } from '@app/query/common/compliance-checker/compliance-checker.query';
+import { useCryptoCurrencyMarketDataMeanAverage } from '@app/query/common/market-data/market-data.hooks';
+import type { StacksAccount } from '@app/store/accounts/blockchain/stacks/stacks-account.models';
+import { useCurrentStacksNetworkState } from '@app/store/networks/networks.hooks';
+
+import { RpcStxTransferSip9NftProvider } from './rpc-stx-transfer-sip9-nft.context';
+import {
+  getDecodedRpcStxTransferSip9NftRequest,
+  getStacksUnsignedContractCallOptionsForFeeEstimation,
+} from './rpc-stx-transfer-sip9-nft.utils';
+
+interface RpcStxTransferSip10FtContainerProps {
+  account: StacksAccount;
+}
+export function RpcStxTransferSip9NftContainer({ account }: RpcStxTransferSip10FtContainerProps) {
+  const request = useRpcTransactionRequest();
+  const network = useCurrentStacksNetworkState();
+  const stxMarketData = useCryptoCurrencyMarketDataMeanAverage('STX');
+  const navigate = useNavigate();
+
+  const rpcRequest = getDecodedRpcStxTransferSip9NftRequest();
+  const txOptionsForFeeEstimation = getStacksUnsignedContractCallOptionsForFeeEstimation({
```

Comment:
```suggestion
  const txOptionsForFeeEstimation = getUnsignedStacksContractCallOptionsForFeeEstimation({
```

this ordering makes more sense to me

Reply:
I think I did it that way bc the type from our Stacks pkg it `StacksUnsignedContractCallOptions`.